mypy_requirements.txt

```diff
@@ -2,4 +2,3 @@
 # These packages are python3 packages to be used by MyPy.
 mypy==0.812
 typed-ast==1.4.2
-types-redis==3.5.4
```

Comment:
Removing this because they are useless to us.
These types-packages will only be detected by mypy when they exist in the environment or globally. So it will be detected only in CI (as we use global installation in CI) not locally. So this can make the stubs for ci and local dev env different. Thanks!